charts/eks-anywhere-packages/templates/deployment.yaml

```diff
@@ -66,6 +66,11 @@ spec:
               value: {{ .Values.proxy.NO_PROXY | quote}}
             - name: CLUSTER_NAME
               value: {{ .Values.clusterName | quote}}
```

Comment:
Why isn't this a file? This doesn't seem to be used.

Reply:
Not sure if REGISTRY_MIRROR in general is a good name, could be a private registry even in ECR